app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -134,6 +107,62 @@ function() {
         return $boltHelper->transmit('orders', $orderRequest);
     }
 
+
+    /**
+     * Get Promise to create an order token
+     *
+     * @param string  $checkoutType  'multi-page' | 'one-page' | 'admin' | 'firecheckout'
+     *
+     * @return string javascript Promise string used for initializing BoltCheckout
+     */
+    protected function getBoltOrderTokenPromise($checkoutType) {
+
+        if ( $checkoutType === self::CHECKOUT_TYPE_FIRECHECKOUT ) {
+            $checkoutTokenUrl = Mage::helper('boltpay/url')->getMagentoUrl('boltpay/order/firecheckoutcreate');
+            $validPreconditionsCheck = 'isFireCheckoutFormValid';
+            $postData = 'checkout.getFormData ? checkout.getFormData() : Form.serialize(checkout.form, true)';
+        } else if ( $checkoutType === self::CHECKOUT_TYPE_ADMIN ) {
+            $checkoutTokenUrl = Mage::helper('boltpay/url')->getMagentoUrl("adminhtml/sales_order_create/create/checkoutType/$checkoutType", true);
```

Comment:
As far as I can tell, the second param for `getMagentoUrl` needs to be an array. Do you mean to have some sort of `['someParam' => true]` here instead?

Reply:
@matt-thomason Good catch.  I missed the second parameter and should have passed an empty array.